### ISSUE TITLE
when reading with nvda + a (notebook layout) the reading does not start from where it left off

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#11987]
### Steps to reproduce:
open the attached document
configure nvda to use the keyboard layout for notebook (laptop computer).
Start reading the document with nvda + a
read at least 3 pages.
press ctrl to stop reading.
press down arrow to read the next line.
### Actual behavior:
the focus is on the first page.
### Expected behavior:
the focus should be on page 3 which is where i pressed ctrl to stop reading.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-21587,98892856

#### Windows version:
10 19042.685
#### Name and version of other software in use when reproducing the issue:
WINWORD.EXE, Microsoft Office 16.0.13426.20404

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If addons are disabled, is your problem still occuring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no
[Apostila Excel Básico 2016 - 30 10 2020.docx](https://github.com/nvaccess/nvda/files/5773163/Apostila.Excel.Basico.2016.-.30.10.2020.docx)

**Comment by fernando-jose-silva** [comment]
other information:
nvda is in focus mode.
I tested it on another document and the result was the same.

**Comment by DGartmann** [comment]
I am having this issue to. Using Word 365 on Windows10 with NVDA 2020.4 beta 3. Has also seen this issue in previous versions of NVDA.

**Comment by bhavyashah** [comment]
CCing @burakyuksek in case they think #11988 may be related or identical to this issue.

**Comment by burakyuksek** [comment]
Yes, my issue is identical to this. I was thinking of closing my issue, but then I saw you made a detailed comment to my issue so I now cannot decide if I should close it or not.
